docs/pvacseq/errors.rst

```diff
@@ -0,0 +1,169 @@
+Common Errors
+-------------
+
+Input VCF Sample Information
+____________________________
+
+**VCF contains more than one sample but sample_name is not set.**
+
+pVACseq supports running with a multi-sample VCF as input. However, it still
```

Comment:
I would avoid using `still` here unless the `sample_name` property is required when the VCF has only one sample, and we are clarifying that it is required in this case, too.
```suggestion
pVACseq supports running with a multi-sample VCF as input. However, in this case it
```